open-craft/ansible-playbooks
# FAL-1752 more setting up for mail server and mailing list

[FAL-1752](https://tasks.opencraft.com/browse/FAL-1752)

Add/update configuration for a mail server that hosts a mailing list (mailman3 + mailman3 web tools), and postfix config for aliases and forwarding mail.  It's relatively simple because this is for our particular use case; this isn't a general 'mail server' or 'mailing list server' role.

**Test instructions**:

- verify that the defaults are sane and documented
- read through the playbook and verify that all the steps make sense, the correct handlers are called, etc.
- check the postfix config

**Reviewers**:
